app/helpers/samples_helper.rb

```diff
@@ -171,12 +173,15 @@ def select_cv_source_ontology(sample_controlled_vocab)
   end
 
   def seek_cv_attribute_display(value, attribute)
-    term = attribute.sample_controlled_vocab.sample_controlled_vocab_terms.where(label:value).last
-    content = value
+    term = attribute.sample_controlled_vocab.sample_controlled_vocab_terms.where(label: value).last
     if term && term.iri.present?
-      content << " (#{term.iri}) "
+      iri_content = term.iri.match?(/^http/) ? link_to(term.iri, term.iri, target: '_blank') : term.iri
```

Comment:
`/^https?:\/\//` would be a bit safer as the regular expression

Reply:
changed to `iri_content = term.iri.match?(/^https?:\/\//) ? link_to(term.iri, term.iri, target: '_blank') : term.iri`